doc/source/developer_guide/dg_forcing.rst

```diff
@@ -24,9 +24,9 @@ use aspects of the following approach,
 - Input files are organized by year.
 - Namelist inputs ``fyear`` and ``ycycle`` specify the forcing year dataset.
 - The forcing year is computed on the fly and is assumed to be cyclical over the forcing dataset length defined by ``ycycle``.
-- The namelist ``atm_dat_dir`` specifies the directory of the atmosphere input data files and the namelist ``atm_data_type`` defines the atmospheric forcing mode.
-- The namelist ``ocn_dat_dir`` specifies the directory of the ocean input data files and the namelist ``ocn_data_type`` defines the ocean forcing mode.
-- The filenames follow a particular naming convention that is defined in the source code (ie. subroutine **JRA55_gx1_files**).  The forcing year is typically found just before the **.nc** part of the filename and there are tools (subroutine **file_year**) to update the filename based on the model year and appropriate forcing year.
+- The namelist ``atm_data_dir`` specifies the directory or a portion of the directory for the atmosphere input data files and the namelist ``atm_data_type`` defines the atmospheric forcing mode.  ``atm_data_type`` values of ``JRA55``, ``JRA55do``, or ``ncar`` provide some flexibility for directory paths and filenames.  These details are best understood by studying the **ice_forcing.F90** source code.
```

Comment:
I've added some suggestions for comments in the code above.  Here, instead of "... provide some flexibility for directory paths and filenames.  These details are best understood...", perhaps something like "... options are provided for testing purposes and use several different data sets provided by the CICE Consortium.  The code automatically accounts for differences in paths and filenames within these data sets; for details or to add new forcing, see the **ice_forcing.F90** source code."